### PR TITLE
SONAR-8938 Remove sonar.search.clusterName in favor of sonar.cluster.name

### PR DESCRIPTION
**Pull request by ehartmann**

### CODE REVIEW

File: server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java [hunk 1]
```diff
@@ -111,7 +110,7 @@ public static void completeDefaults(Props props) {
 
   public static Properties defaults() {
     Properties defaults = new Properties();
-    defaults.put(SEARCH_CLUSTER_NAME, "sonarqube");
+//    defaults.put(CLUSTER_NAME, "sonarqube");
```
Comment (SonarTech) [inline server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: server/sonar-search/src/main/java/org/sonar/search/EsSettings.java [hunk 2]
```diff
@@ -49,8 +52,14 @@
 
   EsSettings(Props props) {
     this.props = props;
-    // name of ES cluster must always be set, even if clustering of SQ is disabled
-    this.clusterName = props.nonNullValue(ProcessProperties.SEARCH_CLUSTER_NAME);
+
+    String clusterName = props.nonNullValue(ProcessProperties.CLUSTER_NAME);
```
Comment (SonarTech) [inline server/sonar-search/src/main/java/org/sonar/search/EsSettings.java]: MAJOR Rename "clusterName" which hides the field declared at line 50. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AHiddenFieldCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: server/sonar-search/src/main/java/org/sonar/search/EsSettings.java [hunk 3]
```diff
@@ -49,8 +52,14 @@
 
   EsSettings(Props props) {
     this.props = props;
-    // name of ES cluster must always be set, even if clustering of SQ is disabled
-    this.clusterName = props.nonNullValue(ProcessProperties.SEARCH_CLUSTER_NAME);
+
+    String configuredClusterName = props.nonNullValue(ProcessProperties.CLUSTER_NAME);
+    if (isBlank(configuredClusterName)) {
```
Comment (simonbrandhof) [inline server/sonar-search/src/main/java/org/sonar/search/EsSettings.java]: condition should be dropped

**Pull request merged**